jdk/make/CompileLaunchers.gmk

```diff
@@ -462,6 +462,10 @@ $(eval $(call SetupLauncher,jps, \
     -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "openj9.tools.attach.diagnostics.tools.Jps" }' \
     -DAPP_CLASSPATH='{ "/lib/tools.jar" }'))
 
+$(eval $(call SetupLauncher,jmap, \
+    -DJAVA_ARGS='{ "-J-ms8m"$(COMMA) "openj9.tools.attach.diagnostics.tools.Jmap" }' \
+    -DAPP_CLASSPATH='{ "/lib/tools.jar" }'))
+
```

Comment:
Can you move this so the launchers remain in alphabetical order?

Reply:
Good idea.  Done.